src/containers/notification/store/sagas.ts

```diff
@@ -433,6 +433,14 @@ function* getNotifications(isFirstFetch: boolean) {
             )
           )
         }
+      } else if (status !== Status.SUCCESS) {
+        yield put(
+          notificationActions.fetchNotificationSucceeded(
+            [], // notifications
+            0, // totalUnread
```

Comment:
this seems like a bad code smell that we would report "success" in the case of non-success. Can we make a new action?